src/list/readme-variants.tsx

```diff
@@ -146,26 +146,46 @@ export default function Readme() {
         Notice the `readOnly` prop is also set on the individual form elements.
       </DocsP>
 
+      
+      <DocsExample label="singleSelection">
+        { function Example() {
+          const [selectedIndex, setSelectedIndex] = React.useState<number>(-1)
+
+          return ( 
+            <List 
+              selectedIndex={selectedIndex} 
+              onAction={ (evt) => setSelectedIndex(evt.detail.index)}
+            ><ListItem>Cookies</ListItem>
+             <ListItem>Pizza</ListItem>
+            <ListItem>Cake</ListItem>
+            </List>
+              )
+          }
+        }
+        </DocsExample>
+
       <DocsExample label="Checkboxes">
         {function Example() {
-          const [checked, setChecked] = React.useState<any>({
-            Cookies: false,
-            Pizza: false,
-            Icecream: false
-          });
+          const shoppingList = ['Cookies','Pizza','Icecream']
+          const [selectedIndex, setSelectedIndex] = React.useState<number[]>([])
+
+          //@ts-ignore
```

Comment:
is ts-ignore need here? What's the typescript error? (I can check out the branch and have a look tomorrow)